types/prettier/index.d.ts

```diff
@@ -0,0 +1,103 @@
+// Type definitions for prettier 1.5
+// Project: https://github.com/prettier/prettier
+// Definitions by: Ika <https://github.com/ikatyang>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+export type AST = any;
```

Comment:
If there is a common AST representation `prettier` uses which is from another package, then it would be useful to import and use that here if it already has types (I know `estree` does, and I think all the parsers must return `estree` trees, looking at the implementation).